source/_integrations/nibe_heatpump.markdown

```diff
@@ -75,4 +80,26 @@ An Arduino-based solution has been tested by OpenHAB community with Arduino uno
 A standard Linux application for running the Nibe Gateway software is also available.
 
 - [Documentation from OpenHAB](https://www.openhab.org/addons/bindings/nibeheatpump/#raspberry-pi-or-other-linux-unix-based-boards)
-- [Source code in OpenHAB Contrib](https://github.com/openhab/openhab-addons/tree/main/bundles/org.openhab.binding.nibeheatpump/contrib/NibeGW/RasPi)
\ No newline at end of file
+- [Source code in OpenHAB Contrib](https://github.com/openhab/openhab-addons/tree/main/bundles/org.openhab.binding.nibeheatpump/contrib/NibeGW/RasPi)
+
+### Modbus Connection
+
+Home Assistant support connecting to Nibe heat pumps over modbus over either TCP or RCU like connections. When connecting to the pump inside a Home Assistant, Modbus connection string must be specified.
+
+#### TCP/IP
+
+The newer S-series pump expose a built in TCP Modbus server that can be used to communicate with the pump. To enable support for this integration, the pump must be connected to your local network and network (5.2 - Network settings) and modbus (7.5.9 - Modbus TCP/IP) must be enabled. More details on the modbus support in the pump can be found in Nibe document [M12676EN](https://partner.nibe.eu/nibedocuments/29795/M12676EN.pdf).
+
+- `tcp://[IP OR HOSTNAME]`
+- `tcp://[IP OR HOSTNAME]:502`
+
+#### RCU
+
+If your system is equipped with a MODBUS40 accessory, this can be used to control your pump. To connect Home Assistant to the MODBUS40 accessory, a RS485 USB dongle is needed, which can be locally connected to you Home Assistant system, or on a separate computer exposed via [ser2net](https://linux.die.net/man/8/ser2net)
```

Comment:
```suggestion
If your system is equipped with a MODBUS40 accessory, this can be used to control your pump. To connect Home Assistant to the MODBUS40 accessory, a RS485 USB dongle is needed, which can be locally connected to your Home Assistant system, or on a separate computer exposed via [ser2net](https://linux.die.net/man/8/ser2net)
```